api/prisma/migrations/08_updating_on_delete_cascades_listings_and_applications/migration.sql

```diff
@@ -0,0 +1,110 @@
+-- DropForeignKey
+
+ALTER TABLE "application_methods"
```

Comment:
These changes aren't referenced in the PR description. But I think I remember Yazeed talking about these changes. These are all just to allow proper deletion of listings?

Reply:
Yep! It's the cascading delete issues.